net.tenshu.Terminator2.yml

```diff
@@ -0,0 +1,47 @@
+---
+id: net.tenshu.Terminator2
+base-version: main
+runtime: org.gnome.Sdk
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: terminator
+rename-desktop-file: terminator.desktop
+rename-appdata-file: terminator.appdata.xml
+rename-icon: terminator
+copy-icon: true
+
+finish-args:
+  - --share=ipc
+  - --socket=session-bus
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=home
+  - --own-name=net.tenshu.Terminator2
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this DBus permission?

Reply:
it's a terminal emulator, it needs to be able to spawn shells on the host itself